style.css

```diff
@@ -0,0 +1,59 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Inter', sans-serif;
+}
+
+body {
+  padding: 24px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.section-1 {
+  background-image: url(img/header-illsutration-mobile.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  width: 100%;
+  height: 100vh;
+}
```

Comment:
- [ ] One of the project requirements is that you use flexbox to style the header and section in this portfolio. Kindly implement this specific project requirement for the section tag in your headline.